spec/helpers/firm_helper_spec.rb

```diff
@@ -54,4 +54,31 @@
       it { is_expected.to eq('2.0 miles away') }
     end
   end
+
+  describe 'minimum_pot_size_text' do
+    let(:available_ordinals) { I18n.t('investment_size.ordinal').keys.map(&:to_s).map(&:to_i) }
```

Comment:
Changed the definition of `let(:other_size)` so we don't depend on the order (within the test).